scripts/drift_corr_ion.py

```diff
@@ -0,0 +1,155 @@
+"""
+Created on
+
+@author:
+
+Copyright © , Delmic
```

Comment:
Hello Eric do I need to add these details? If yes, what would be the entries here?

Reply:
Yes, put your name there :-)